### DISCUSSION TITLE
Tasmota+ADS 1115

### DISCUSSION THREAD

**Discussion by wadim-tim** [arendst/Tasmota#20224] [Q&A] [answered]
Hello. Is it possible, when connecting two or more ADS1115s, to select differential mode for one and single-sided mode for the other?

**Comment by sfromis** [comment, marked as answer]
No, as indicated in the syntax for the relevant [Sensor12](https://tasmota.github.io/docs/Commands/#sensor12) command, the driver does not provide a way of selecting modes or ranges by channel or even by ADS1115 chip.
